src/Behat/Transliterator/data/x22.php

```diff
@@ -1,19 +0,0 @@
-<?php
-$UTF8_TO_ASCII[0x22] = array(
```

Comment:
When I was initially syncing char tables I noticed that several files were introduced to git, they were all containing only `[?]` inside. I thought as those tables contain no information, they were redundant for PHP implementtion. But in PHP data you still have some files containing only `[?]`. So the overall policy is inconsistent. If tables of only `[?]` are needed, I can enable them back, but this will produce some new files. If they are not needed, some existing files should be deleted.

Please share your knowledge on this topic.